liquidity-module/index.md

```diff
@@ -256,6 +264,23 @@ See your balance at [https://api.testnet.cosmos.network/cosmos/bank/v1beta1/bala
 
 Take a closer look at the `ibc/denomhash` denominator. When you create a new pool, you in put this denom to make a pair with one of the existing native token. On the testnet, create a pair with `uphoton`.
 
+**Tip:** See the balance of `uphoton` and `ibc/denomhash` on terminal:
+```bash
+gaiad query bank balances cosmosxxxx --node https://testnet.cosmos.network:443
+```
+
+After you successfully query the balance, you will see an output similar to:
+```markdown
```

Comment:
I always wonder what to put here for syntax annotation, is "markdown" the correct syntax? what about "bash" ? or "text"?

Reply:
We could. I don't think it makes a huge difference though.